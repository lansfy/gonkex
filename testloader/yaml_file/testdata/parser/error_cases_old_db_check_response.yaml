- name: loader MUST fail if cannot substitute dbResponseArgs to query response
  method: GET
  path: /some/path
  response:
    200: "aaabbb"
  dbQuery: "SELECT ALL"
  dbResponse:
    - "{{ .bar } item1"
    - "{{ .bar }} item2"
  cases:
    - dbResponseArgs:
        bar: value1
