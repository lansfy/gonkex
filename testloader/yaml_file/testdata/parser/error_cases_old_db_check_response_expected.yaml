- Error: 'preprocess file testdata/parser/error_cases_old_db_check_response.yaml: template: :1: unexpected "}" in operand'
